articles/vlan-trunking-and-encapsulation/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: vlan-trunking-and-encapsulation
+title: Overview of VLAN Trunking and Encapsulation
+description: This article briefly introduces Virtual Local Area Networks(VLANs) and their requirements. It discusses VLAN trunking and VLAN encapsulation using IEEE 802.1Q standard.
+author: shreya-a-n
+date: 2020-08-01T00:00:00-12:00
+topics: [networking]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/vlan-trunking-and-encapsulation/hero.jpg
+    alt: vlan trunking and encapsulation
+
+---
+### What is a VLAN?
+**A [virtual LAN (VLAN)](https://en.wikipedia.org/wiki/Virtual_LAN) is any broadcast domain that is partitioned and isolated in a computer network at the data link layer (OSI Layer 2).** LAN stands for Local Area Network and virtual, in the name, upholds the concept of separating broadcast domains of networks logically. *A broadcast domain is a network that is logically divided such that all nodes in the network can reach each other by broadcast at the data link layer*. *VLANs function by tagging every frame with a VLAN tag and designing the network such that it takes these tags into account while performing routing and forwarding.* By assigning tags to frames one can create the appearance and functionality of a computer network that is physically on a single network but acts as if it is split between separate networks. In simple terms, VLAN segregates traffic within a network.
+
+### What is VLAN trunking?
+There are two ways of categorizing VLAN enabled ports on network devices:
+- Untagged or Native ports
+- Tagged or Trunk ports
+
+ *A network port is a communication endpoint a device uses to send and receive traffic.*
+
+**Native ports accept traffic from a single VLAN and are used to link end devices like routers, servers, terminals etc.** They do not need to accommodate an additional identification tag as they are involved in exchanging traffic with a single end device. Hence the name, **untagged ports**. They are also referred to as **access ports.**
+**Trunk ports are used to pass traffic from multiple VLANs and are used to link switches.** In order to distinguish between frames from different VLANs, trunk ports add identification tags to the frames.
+Hence the name, **tagged ports.**
+**A trunk is a point-to-point link between two network devices that carry more than one VLAN.**
+
+![native-vs-trunk](/engineering-education/vlan-trunking-and-encapsulation/native-vs-trunk.jpg)
```

Comment:
@ShreyaAN10 wanted to ask if you could include the image sources underneath the images

Reply:
@hectorkambow , I didn't pick these images from the web. I made them using diagrams.net